server/controllers/topicPage.controller.js

```diff
@@ -1,4 +1,5 @@
 import mongoose from 'mongoose';
+import every from 'lodash/every';
```

Comment:
Not been used?

Reply:
Actually, I'm using it on line 14, but this did help me catch a different issue. Line 14 should be `&&` instead of `II` — thanks!